scripts/appscript.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC2154
```

Comment:
Why disable this one?

Reply:
this is terraform templated so thought i'd be easier to future proof it here rather than at each var. I can specify each var if that's preferred though

Reply:
i think so, yes. not a fan of wholly disabling checks, except as a last resort. might also help us keep the templated vars defined near the top of the file.

Reply:
rgr updated